src/lib/User/actions.ts

```diff
@@ -130,3 +134,14 @@ export async function signupUser(
 
   return redirect("/");
 }
+
+export async function signoutUser() {
```

Comment:
Should this be "async"? I don't see any use of "await" inside.

Reply:
actions can only export async functions, I think.

Reply:
Right! That makes sense lol